python/mkdocs/docs/api.md

```diff
@@ -28,6 +28,19 @@ catalog:
     credential: t-1234:secret
 ```
 
+These info must be placed inside a file called `.pyiceberg.yaml` located either in the `$HOME` directory or in the `$PYICEBERG_HOME` directory (if var is set).
+
+Examples of other values:
+
+| Config key             | Description                     | [Example](https://github.com/tabular-io/docker-spark-iceberg) |
```

Comment:
This information is under the configuration page: https://py.iceberg.apache.org/configuration/

This PR shows that it wasn't obvious, should we refer to that page?

Reply:
Hi @Fokko, thanks for your review :). Yes you are right, I have updated the PR, let me know what you think. Thanks!